Frends.LDAP.AddUserToGroups/Frends.LDAP.AddUserToGroups.Tests/UnitTests.cs

```diff
@@ -0,0 +1,130 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Frends.LDAP.AddUserToGroups.Definitions;
+using Novell.Directory.Ldap;
+namespace Frends.LDAP.AddUserToGroups.Tests;
+
+[TestClass]
+public class UnitTests
+{
+    // Azure AD.
+    /*
+        private readonly int _port = 389;
+        private readonly string? _path = "CN=Users,DC=FRENDSTest01,DC=net";
+        private readonly string? _dn = "CN=Tes Tuser,CN=Users,DC=FRENDSTest01,DC=net";
+        private readonly string? _groupDn = "CN=Users,CN=Builtin,DC=FRENDSTest01,DC=net";
+        private readonly string? _host = Environment.GetEnvironmentVariable("HiQ_AzureADTest_Address");
+        private readonly string? _user = Environment.GetEnvironmentVariable("HiQ_AzureADTest_User");
+        private readonly string? _pw = Environment.GetEnvironmentVariable("HiQ_AzureADTest_Password");
```

Comment:
Since tests are run in a container, these can be removed.

Reply:
These are left here for easier debugging in the future. It's easier to view results in Azure AD than in LDAP dummy server. DNs and other parameters aren't the same in Azure and dummy server.